code/obj/machinery/shield_generators/shield_generator.dm

```diff
@@ -130,10 +130,9 @@
 		var/diff = world.timeofday - lastuse
 		if(diff < 0) diff += 864000 //Wrapping protection.
 
-
 		if(diff > 1500)
 			lastuse = world.timeofday
-			visible_message("You flip the switch on [src].","You feel around the object and accidentally hit a switch")
+			visible_message("[src] makes an annoyed beep.","You hear an annoyed beep.")
```

Comment:
why would the machine be annoyed that you're toggling it? this sounds like it would be a error message, not a success message

Reply:
12:30 am brain was annoyed thats why, ill change that